.travis.yml

```diff
@@ -8,30 +8,26 @@ cabal: "3.0"
 
 cache:
   directories:
-  - "$HOME/.cabal/store"
-  - "$HOME/.stack"
-  - "$TRAVIS_BUILD_DIR/.stack-work"
+    - "$HOME/.cabal/store"
+    - "$HOME/.stack"
+    - "$TRAVIS_BUILD_DIR/.stack-work"
 
 matrix:
   include:
+    # Cabal
+    - ghc: 8.6.5
```

Comment:
I guess now it becomes much easier to support earlier GHCs, so let's not remove it until it becomes difficult to maintain :slightly_smiling_face: